app/jobs/split_pdf_job.rb

```diff
@@ -236,11 +238,96 @@ def save_pages(exam_template, partial_exams, filename=nil, split_pdf_log=nil)
                 'application/pdf')
         end
         repo.commit(txn)
+
+        # convert PDF to an image
+        imglist = Magick::Image.from_blob(cover_pdf.to_pdf) do
+          self.quality = 100
+          self.density = '300'
+        end
+
+        img = imglist.first
+        # Snip out the middle of PDF that contains the student information
+        # TODO: store coordinates in exam template instead of hardcoding
+        student_info = img.crop 50, 1100, img.columns, img.rows / 6.0
+        student_info_file = File.join(raw_dir, "#{grouping.id}_info.jpg")
+        student_info.write(student_info_file)
+
+        begin
+          out = `python3 lib/scanner/read_chars.py #{student_info_file}`
+          tokens = out.split("\n")
```

Comment:
Do some error-checking after this. If tokens doesn't have the correct length execute `next` to skip.